### PR TITLE
Configure: Filtering and Tags

### PR DESCRIPTION
**Pull request by mohameddahrouj**
Consolidated the views for configuring filters and tags (permission levels/roles) under a configure view.

i.e.:
image

### CODE REVIEW

File: SECP-service/src/main/resources/assets/app/views/portal/configure-tags.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<p class="note"><strong>Note:</strong> Add and delete permission levels and user roles by clicking below.</p>
+<div>
+    <br>
+    <p class="tag-option"><a class="configureoption" ng-click="permissionClick()"><span class="optionicon" id="optionicon1">+</span><i class="fa fa-low-vision" aria-hidden="true"></i> Permission Levels</a></p>
+    <hr id="optionline1">
+    <br>
+    <p class="tag-option"><a class="configureoption" ng-click="rolesClick()"><span class="optionicon" id="optionicon2">+</span><i class="fa fa-id-card-o" aria-hidden="true"></i> Roles</a></p>
+    <hr id="optionline2">
+</div>
+<div id="permissionTableDiv" style="display: none;">
```
Comment (lavatahir) [inline SECP-service/src/main/resources/assets/app/views/portal/configure-tags.html]: Would be cool if you put the display: none in css file but doesnt matter

**Comment by lavatahir** [comment]
why is codecov complaining?

File: SECP-service/src/main/resources/assets/app/views/portal/configure.html [hunk 2]
```diff
@@ -1,29 +1,10 @@
...
-    <!-- Role Modal -->
-    <secp-modal ng-model="roleInput" name="roleModal" frmName="newroleform" title="New Role"
-                body="Role" save-fn="saveRole()"></secp-modal>
+<h1 class="portalheader">
+    <i class="fa fa-wrench" aria-hidden="true"></i>
+    Configure
+</h1>
+<div>
+    <a href="/portal/configure/filter"><p class="adminoption"><i class="fa fa-filter" aria-hidden="true"></i> Filter</p></a>
+    <a href="/portal/configure/tags"><p class="adminoption"><i class="fa fa-tags" aria-hidden="true"></i> Tags</p></a>
```
Comment (tosinoni) [inline SECP-service/src/main/resources/assets/app/views/portal/configure.html]: Is there a reason why you chose the word Tags? I'm not sure how it relates to roles and permissions.

Comment (mohameddahrouj) [reply SECP-service/src/main/resources/assets/app/views/portal/configure.html]: I was trying to think of an appropriate word for both. If anyone has another suggestion let me know.

File: SECP-service/src/main/resources/assets/app/views/portal/configure-tags.html (hunk 1 above)
Comment (mohameddahrouj) [reply SECP-service/src/main/resources/assets/app/views/portal/configure-tags.html]: Agreed ✔️

**Pull request merged**